crates/bevy_camera/src/camera.rs

```diff
@@ -614,19 +614,23 @@ impl Camera {
         // Flip the Y co-ordinate origin from the top to the bottom.
         rect_relative.y = 1.0 - rect_relative.y;
 
-        let ndc = rect_relative * 2. - Vec2::ONE;
-        let ndc_to_world = camera_transform.to_matrix() * self.computed.clip_from_view.inverse();
-        let world_near_plane = ndc_to_world.project_point3(ndc.extend(1.));
-        // Using EPSILON because an ndc with Z = 0 returns NaNs.
-        let world_far_plane = ndc_to_world.project_point3(ndc.extend(f32::EPSILON));
+        let ndc_point_near = (rect_relative * 2. - Vec2::ONE).extend(1.0).into();
+        let view_point_near = self
```

Comment:
```suggestion
        // We multiply the point by `view_from_clip` and then `world_from_view` in sequence to avoid the precision loss
        // (and performance penalty) incurred by pre-composing an affine transform with a projective transform.
        // Additionally, we avoid adding and subtracting translation to the direction component to maintain precision.
        let view_point_near = self
```